### PR TITLE
Add recipe for serialchemy

### PR DESCRIPTION
**Pull request by lincrosenbach**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by igorcandeia** [comment]
Good job!

File: recipes/serialchemy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  license_file: LICENSE
+  summary: Serializers for SQLAlchemy models.
+  doc_url: https://serialchemy.readthedocs.io/
+  dev_url: http://github.com/ESSS/serialchemy
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - igortg
+    - kfasolin
```
Comment (kfasolin) [inline recipes/serialchemy/meta.yaml]: I'm ok being a maintainer 👍

File: recipes/serialchemy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - nicoddemus
```
Comment (nicoddemus) [inline recipes/serialchemy/meta.yaml]: OK for being a mainteiner. 👍

File: recipes/serialchemy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - sqlalchemy >=1.1
```
Comment (nicoddemus) [inline recipes/serialchemy/meta.yaml]: You don't need this during build time, you don't import `sqlalchemy` in `setup.py`. 👍

File: recipes/serialchemy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: Serializers for SQLAlchemy models.
+  doc_url: https://serialchemy.readthedocs.io/
+  dev_url: http://github.com/ESSS/serialchemy
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - igortg
```
Comment (nicoddemus) [inline recipes/serialchemy/meta.yaml]: @igortg is on vacation and probably won't be able to answer, so I suggest you remove it for now and we can add him later.

File: recipes/serialchemy/meta.yaml (hunk 3 above)
Comment (lincrosenbach) [reply recipes/serialchemy/meta.yaml]: Removed

File: recipes/serialchemy/meta.yaml (hunk 4 above)
Comment (lincrosenbach) [reply recipes/serialchemy/meta.yaml]: Drop @igortg from maintainers for now

**Comment by lincrosenbach** [comment]
> A tip: for `noarch: python` packages add `[skip appveyor] [skip travis]` to the first line of the commit to avoid triggering those CI providers; this will get the job green sooner.

Made a commit ammend, hope it works.

**Comment by marcelotrevisani** [comment]
LGTM!

**Pull request merged**